docs/builtins/Time/parse-time.md

```diff
@@ -0,0 +1,32 @@
+## parse-time
+Use `parse-time` to construct time from a UTC value using a specified format. See ["Time Formats" docs](pact-reference.html#time-formats) for supported formats.
```

Comment:
this is an invalid link.
please change to `https://docs.kadena.io/reference/functions/time#time-formatsh-299815639`